server/routes/payment.js

```diff
@@ -0,0 +1,110 @@
+var express = require('express');
+var router = express.Router();
+//Testing Paypal API
+require('dotenv').config()
+let profile_name = Math.random().toString(36).substring(7);
+let paypal = require('paypal-rest-sdk')
+
+//We may have to store USER_ID for name to prevent the creation of new profiles
+//To prevent the possibility for an error thrown in the creation of one
+const web_profile = {
+    "name": profile_name,
+    "presentation": {
+        "brand_name": "Bright Futures Academy",
+        "locale_code": "US"
+    },
+    "input_fields": {
+        "allow_note": false,
+        "no_shipping": 1,
+        "address_override": 1
+    }
+}
+
+
+
+paypal.configure({
+    'mode': 'sandbox',
+    'client_id': 'AZPizo1pg8Ne0XFiy5aaO_fp4Q65kKzcDDPLvX4gWmG_5EaFrhk-BEuYsSYESi2c-RwFYjA1sBxQe4fS',
+    'client_secret': 'ECNjD9CVuKfojZxyQZL934P12Wbkv8HPQHEWI___ijt6Xy5DtKmafkTV5s1fckEvDbc0JeTQUIwylNoz'
```

Comment:
We definitely need to put these into a .env file so that they aren't leaked. We would put these in a file on Heroku ourselves.